cli-plugins/manager/metadata.go

```diff
@@ -25,4 +25,6 @@ type Metadata struct {
 	// Experimental specifies whether the plugin is experimental.
 	// Deprecated: experimental features are now always enabled in the CLI
 	Experimental bool `json:",omitempty"`
+	// Completions specifies whether the plugin supports command completion
+	Completions bool `json:",omitempty"`
```

Comment:
Perhaps we should consider updating the SchemaVersion as well

```
// SchemaVersion describes the version of this struct. Mandatory, must be "0.1.0"
```

Do you think we can combine this PR with https://github.com/docker/cli/pull/3254 ? (feel free to take that branch of course, if there's useful bits in it)

Reply:
I'm not sure about #3254. I understand we want to keep backward compatibility for users, but on the other hand using cobra completion v2 bring more flexibility and CLI plugins support, so should be preferred. If we support both, we will have to offer double maintenance...